zstacklib/zstacklib/utils/multipath.py

```
@@ -65,34 +79,50 @@ def write_multipath_conf(path, blacklist=None):
         if not has_devices_section:
-            config.append({'devices': [default_device]})
-            modified = True
+            self.config.append({'devices': [default_device]})
+            self.modified = True
+
+
+    def config_section(self, name, cfg):
+        for section in self.config:
+            if name not in section:
+                continue
+            elif cmp(sorted_conf(section[name]), sorted_conf(cfg)) == 0:
```

Ruff (0.12.2)

Undefined name `cmp`

(F821)